hw/top_earlgrey/doc/dv_plan/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Earlgrey Chip DV Plan"
+---
+
+## Goals
+* **DV**
+  * Verify `top_earlgrey` features by running dynamic simulations with a SV/UVM based testbench.
+  * Verify the integration of all pre-verified IPs instantiated in the chip.
+  * Verify the integration and the internal design of un-verified IPs instantiated in the chip.
+  * Verify system level scenarios for correctness of our design assumptions and behavior.
+  * Verify the full chip configuration and memory address space by running the automated tests.
+  * Stress test the XBAR structures in the chip.
```

Comment:
might want to mention what the coverage goal for top-level testing might be. All IO toggle coverage?

Reply:
Ohh, this will require a more detailed description and a diagram to explain it. I will make a separate PR for this.